### PR TITLE
Config change for fax

### PR DESCRIPTION
**Pull request by tingaloo**
I rule and have completed some work on Case Manager that's ready for review!

I don't really like the design pattern I used here, primarily because there's both a service and url portion. Config is a little bloated, could use some thinning.

This pull request makes the following changes:
* Fax env var to config.

(If there are changes to the views, please include a screenshot so we know what to look for!)

It relates to the following issue #s: 
* Bumps #1495

### CODE REVIEW

File: app/views/layouts/_footer.html.erb [hunk 1]
```diff
@@ -6,11 +6,12 @@
 		</div>
 	</div><!-- end row -->
 </div><!-- end container -->
-	  
+
 <div id="app_footer" class="container text-center margin-bottom-sm">
   <a href="http://<%= FUND_DOMAIN %>/"><%= FUND_FULL %></a> -- <a href="tel:<%= FUND_PHONE %>"><span class="glyphicon glyphicon-earphone" aria-hidden="true"></span> <%= FUND_PHONE %></a><br>
 	<a href="https://prochoice.org/">National Abortion Federation</a> -- <a href="tel:800-772-9100"><span class="glyphicon glyphicon-earphone" aria-hidden="true"></span> 800-772-9100</a><br>
-	<%= link_to FUND_FAX_SERVICE, "http://#{FUND_FAX_SERVICE}", target: '_blank', rel: 'noopener nofollow' %>
+	<a href="http://<%= fax_service %>/"><%= fax_service %></a> -- <a href="tel:<%= fax_number %>"><span class="glyphicon glyphicon-earphone" aria-hidden="true"></span> <%= fax_number %></a><br>
```
Comment (colinxfleming) [inline app/views/layouts/_footer.html.erb]: Rails has a pretty sick helper for cleaning up this kind of thing - https://api.rubyonrails.org/v5.2.1/classes/ActionView/Helpers/UrlHelper.html#method-i-link_to

So we can effectively replace this with `link_to nil, fax_service, target:'_blank', rel: 'noopener nofollow'` and it'll save us having to do http etc.

```suggestion
	<%= link_to nil, fax_service, target: '_blank', rel: 'noopener nofollow' %>
```

File: app/models/config.rb [hunk 2]
```diff
@@ -41,6 +41,14 @@ def options
     config_value[:options]
   end
 
+  def fax_service
+    config_value[:service]
+  end
+
+  def fax_number
```
Comment (colinxfleming) [inline app/models/config.rb]: You know, I think you're absolutely right on how the fax number here makes things much more complex. Let's ditch it and just go with fax service, and ask them to make that a url like `http://whatever.com`. Hopefully that cuts down on a little bit of the complexity/ugmo-ness. That way we can make it handle more like `resources_url`. thoughts?

**Comment by tingaloo** [comment]
@vraravam Hey, i'd like to hand this issue over to you if you'd like. I don't have as much domain knowledge on the performance aspects.

**Comment by vraravam** [comment]
@tingaloo - I'm going to create a new PR with all of my changes - will attribute some of the work to you (but, I dont feel comfortable committing on top of yours due to lots of differences in opinion).

**Comment by colinxfleming** [comment]
I think a takeaway I have is that our config model is starting to break a little bit. We were previously only really using it for drop down options, and it doesn't apply as well to singleshot stuff like this. Not something we have to immediately resolve but it's def a little too magical in a weird way right now.

**Pull request merged**